packages/bitcore-p2p-doge/package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "bitcore-p2p-doge",
+  "version": "8.24.2",
+  "description": "Interface to the bitcoin P2P network for bitcore",
+  "main": "index.js",
+  "scripts": {
+    "test": "gulp test",
+    "test:ci": "npm run test",
+    "coverage": "gulp coverage",
+    "build": "gulp",
+    "pub": "npm run build && npm publish"
+  },
+  "keywords": [
+    "bitcoin",
+    "bitcore"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/bitpay/bitcore-p2p.git"
```

Comment:
I think we can get rid of this "repository" node. Or update the link to https://github.com/bitpay/bitcore/tree/master/packages/bitcore-p2p-doge"?,